composer.json

```diff
@@ -97,6 +97,7 @@
         "symfony/twig-bundle": "4.2.* || 4.3.*",
         "symfony/web-profiler-bundle": "4.2.* || 4.3.*",
         "symfony/yaml": "4.2.* || 4.3.*",
+        "terminal42/service-annotation-bundle": "dev-master",
```

Comment:
Before this PR can be merged, you will have to release a stable version of the bundle.

Reply:
Done in 274efb7e90e0159ca438feda22446adef6a5921a